core/domain/prod_validators.py

```diff
@@ -46,17 +46,16 @@
 
 (
     base_models, collection_models,
-    email_models, exp_models, feedback_models,
-    job_models, question_models, skill_models, story_models,
-    subtopic_models, suggestion_models, topic_models,
-    user_models
+    email_models, exp_models, feedback_models, 
```

Comment:
Remove unwanted space from the end of this line*

Reply:
Done!